### PR TITLE
Display emojis with test results

### PR DESCRIPTION
**Pull request by joelostblom**
- This PR adds emojis for individual test that pass or fail, as well as if all tests pass. I think the emojis for individual tests makes it easier to quickly spot what is right and what is wrong, and it gives a certain satisfaction to see many green checkmarks (personal opinion of course).
- Although the emojis are green and red, I try to choose sufficiently different shapes to not be an issue for people with color vision deficiencies.
- The addition of the failure message will also facilitate implementing #419 where the traceback is optionally removed.
- The random positive emoji when all tests passes is similar to what `test_that` in R does and is more of a fun touch.

Individual tests:

image

All tests:

image

image

### CODE REVIEW

File: otter/test_files/ok_test.py [hunk 1]
```diff
@@ -90,7 +90,9 @@ def run(self, global_environment):
         for i, test_case in enumerate(self.test_cases):
             passed, result = run_doctest(self.name + ' ' + str(i), test_case.body, global_environment)
             if passed:
-                result = 'Test case passed!'
+                result = '✅ Test case passed'
+            else:
+                result = '❌ Test case FAILED\n' + result
```
Comment (chrispyles) [inline otter/test_files/ok_test.py:95]: I don't think the caps is necessary in `FAILED` since we've got the emoji

**Review by chrispyles — CHANGES_REQUESTED** [review]
Can you also look at adding the emojis to the new exception-based test files? I think editing `message` in the `run()` method should be sufficient: https://github.com/ucbds-infra/otter-grader/blob/master/otter/test_files/exception_test.py#L156

**Comment by chrispyles** [comment]
@joelostblom can you also add an entry to the changelog?

File: otter/test_files/ok_test.py (hunk 1 above)
Comment (joelostblom) [reply otter/test_files/ok_test.py]: I was on the edge about this too, changed to lower case now.

**Comment by joelostblom** [comment]
> Can you also look at adding the emojis to the new exception-based test files? 

@chrispyles I had a go at doing this the same way as for the regular ok-tests, but there seem to be not test files in otter CI to check exception based tests so not sure I have gotten it right. I am not familiar with this new format of tests and I am not clear on the differences versus the old format after reading the docs, so I might need some help if there are significant other changes needed.

**Review by chrispyles — APPROVED** [review]

**Comment by chrispyles** [comment]
> but there seem to be not test files in otter CI to check exception based tests so not sure I have gotten it right

+1. I need to add tests for this behavior.

**Pull request merged**